eZ Platform/Studio/Form Builder/Deleting Form/deleting_a_form.md

```diff
@@ -0,0 +1,92 @@
+#Form Builder M3 - Deleting a Form
+
+## Introduction
+This document describes how an Editor can delete a form from the Form Manager's user interface.
+
+
+## Functional Specifications -  User stories
+
+
+### User Stories
+
+User Story ID | User Story Description
+:------------ |:-------------
+1             | ``` As a User, I should have the ability to select a form in the Form Manager for deletion ```
+2             | ``` As a user, I should not be allowed to delete a form if the selected form is in use on the landing page ```
+3             | ``` As a user, I should have the option to navigate to the landing page (in EDIT mode) when I try to delete a form in use on the landing page```
+4				 | ``` As a user, I should be warned when deleting a form in the Form Manager if the form contains collected data```
+5				 | ``` As a user, I should have the option to download collected data when I try to delete a form which contains collected data```
+6             | ```As a user, I should be warned when deleting a form in the Form Manager```
+
+
+
+
+## Functional Specifications -  Detailed behaviors
+
+### User story 1 - As a User, I should have the ability to select a form in the Form Manager for deletion
+*Changes to the current implementation include: <br /> 1. adding a radio button for each form in the table and <br /> 2. adding a 'Delete' button at the top right corner of the table. <br /><br /> By default (if forms exists), then no form will be selected and the 'Delete' button will be disabled.*
+
+***Please refer to the attached image 'FormManagerwithDeleteOption.png'***
```

Comment:
You can embed the image into the markdown, it improves the readability.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

So for this image, it can look like: `![](FormManagerwithDeleteOption.png)`